src/server/pegasus_server_impl.cpp

```diff
@@ -68,6 +68,7 @@ static bool chkpt_init_from_dir(const char *name, int64_t &decree)
 std::shared_ptr<rocksdb::RateLimiter> pegasus_server_impl::_s_rate_limiter;
 int64_t pegasus_server_impl::_rocksdb_limiter_last_total_through;
 std::shared_ptr<rocksdb::Cache> pegasus_server_impl::_s_block_cache;
+std::shared_ptr<rocksdb::WriteBufferManager> pegasus_server_impl::_s_write_buffer_manager;
```

Comment:
what about unique_ptr?

Reply:
In include/rocksdb/options.h, it is defined as std::shared_ptr<WriteBufferManager> as a member of rocksdb::DBOptions, so we'd better keep shared_ptr here ?